src/auth/user-import-builder.ts

```diff
@@ -370,6 +376,25 @@ export class UserImportBuilder {
     return importResult;
   }
 
+  /**
+   * Checks whether is a hashInputOrder-supported hashing algorithm.
+   * @param {HashAlgorithmType | undefined} hashAlgorithmType Hash algorithm type.
```

Comment:
Do you mean for this function only? 
It's been a long time, but I remember I probably got an error for removing undefined or the whole thing.

Reply:
Do this for all the functions you introduced. It won't complain about javadocs but it will complain if the type definitions are incorrect. The explicit types were needed when we couldn't easily determine them. In typescript, they are explicitly defined in the function definition.
The TS style guide is a [helpful reference](https://g3doc.corp.google.com/javascript/typescript/g3doc/styleguide.md?cl=head).